### ISSUE TITLE
Colors Survey!

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6463]
**Brief summary**
Community contributor @hackerncoder has [created a survey](https://github.com/QubesOS/qubes-issues/issues/6414#issuecomment-798912113) (thank you!) to learn more about how folks use colors, in Qubes. This Issue is to contain feedback and findings to/from that effort. It compliments Issue #2523 and originated on #6414.

That survey is here:
https://survey.encryptionin.space/index.php?r=survey/index&sid=178339&lang=en

**Comment by ninavizz** [comment]
From @hackerncoder:

> @ninavizz Here is my survey: https://survey.encryptionin.space/index.php?r=survey/index&sid=178339&lang=en. I would love some feedback (I'm a developer, not a UX person) before I send it out.

**Comment by ninavizz** [comment]
@hackerncoder Thank you for reaching-out and for putting this together! Sure, a few thoughts...

1. You want to ask as little as possible of people, when hitting them up with a survey. Asking them to "think" is often a showstopper... even though, yes, as researchers, that is basically our motivation—to learn how they think (lol). As such, fixed-choice options with easy to parse answers, will more often be answered, than open-text questions. Likewise, fixed-choice questions are easier to meaningfully analyze. 

2. LimeSurvey has a bug w/ some version of Firefox and Android. On our survey, I included a warning to folks on the first page. I'd recommend you do the same... especially since we got folks investing a LOT of time into answers, then feeling immense frustration when their browsers cut them off and dumped their surveys. 

3. I'd also love it, if you could qualify your survey as coming from a community member, and not the Qubes team itself... for a few reasons. One, survey fatigue: this ain't us, it's you! :) Two, I want folks to be clear who is owning this data—it ain't us. Including a link to your website, or your GH, I'd also advise—just to be kosher and transparent.

Finally, I'd recommend creating two sections for your survey. The first, should be fixed-answer options. The second, should be the free-text options. People will more likely answer the prior than the latter—and they well could have more meaningful answers to contribute to one or the other. 

I pulled 2 questions from our existing "Welcome!" survey, and drafted 2, for the below, as my suggestions for the first section. My syntax below is radio-buttons for parenthesis, and checkboxes for brackets. 

1. How do you utilize the VM colors in qubes?
( ) To 'theme' different isolation domains
( ) To indicate different threat levels
( ) I haven't given it any thought
( ) I haven't decided, yet
( ) Other: please describe, below [text field to describe] 
  

2. Do you feel you have too few or too many color choices, for how you use the qube colors?
( ) Too few
( ) Too many
( ) Neither—but different palettes for the 7 color options would be nice to pick from
( ) Other: [open text field to explain]


3. How long have you been using Qubes OS?
( ) Less than 6mos
( ) Between 6mos and 2 years
( ) Between 2 and 4 years 


4. Which of the following traits describe you?
[ ] Academic 
[ ] Activist
[ ] Business Owner
[ ] Employer
[ ] Executive
[ ] FOSS/Libre project contributor
[ ] Information or IT security professional
[ ] IT generalist professional
[ ] Journalist
[ ] Non-Profit/NGO worker
[ ] Private sector (non-government) worker
[ ] Software or web developer
[ ] Student
[ ] Other: [textfield]

**Comment by hackerncoder** [comment]
Thank you nina! As I said, I'm a dev. I have added all of what you suggested.

**Comment by ninavizz** [comment]
WOO!!! Really looking forward to seeing the results... TY again, for doing this! <3 <3 <3

**Comment by ninavizz** [comment]
BTW @hackerncoder there is an option in Lime to _not_ datestamp answers... which actually does datestamp them, just all to the same datestamp in 1983 or something. You may want to turn that on, as an extra layer of security for yourself. FYI. :)

**Comment by hackerncoder** [comment]
I think the datestamp is just whether the survey is anonymous (mine is): https://forums.limesurvey.org/forum/design-issues/84287-adjusting-date-time-stamp. I also checked the answers, and they are dated 1980-01-01...

I'm also excited, 3 complete responses as of now. 6 that have done the first page.

**Comment by hackerncoder** [comment]
I think this is just how it will be, so it should just be creating a post on the forum, a mail to qubes-users and a post on reddit right?

**Comment by andrewdavidwong** [comment]
> I think this is just how it will be, so it should just be creating a post on the forum, a mail to qubes-users and a post on reddit right?

It depends on what exactly you're trying to do, but that sounds right.

**Comment by hackerncoder** [comment]
Spreading the word that I have created a survey and hopefully getting people to do it.

**Comment by andrewdavidwong** [comment]
> Spreading the word that I have created a survey and hopefully getting people to do it.

Yep, those are the places to spread the word. One thing to keep in mind is that messages sent to qubes-users are mirrored read-only onto the forum.

**Comment by ninavizz** [comment]
My only advisement would be to keep the post with the link to the survey short, and to be mindful of the first sentence. The way we phrased the calls on the Qubes download page, I really like. 
- First 1-2 sentences "Hey, take a few minutes to do this thing! It's anonymous, etc" 
- Then the link
- Then, explain what you're hoping to learn and why it matters
- Then share when you expect to share-out "findings," aka results.

Which channels you post the above to, entirely your call. I'm personally unable to keep my head above water in the forums and email lists, so if you feel up to navigating those waters, extra w00t! Go forth and learn, we be cheerin' for ya (and of course, excited to learn what you learn!). TY, again!

**Comment by hackerncoder** [comment]
I would post to all three, but reddit isn't letting me, and my qubes-users mail doesn't seem to have landed? Anyways, I'll figure it out.

I have again added all of your suggestions.

I have already learned a few things, but it is of course waay to early to say anything.

**Comment by ninavizz** [comment]
It's REALLY hard to resist running forth with the excitement "WOO, I learned these things and now I know!" after your first few answers. That will never go away, but I'm thrilled you're doing this new thing! TY for inviting me/us along for the ride. :)

**Comment by andrewdavidwong** [comment]
> How long have you been using Qubes OS?
>  Choose one of the following answers
> Less than 6 months
> Between 6 months and 2 years
> Between 2 and 4 years
> No answer

FYI, there is no option for people like me who have been using Qubes for more than four years, so I just selected "Between 2 and 4 years." :slightly_smiling_face: 

(If that's what you intended for that option, then you could simply reword it to "More than two years.")

**Comment by ninavizz** [comment]
Well, crud! I need to update our survey to add a "+4yrs" entry. 
@hackerncoder COuld you perhaps add a "+4yrs" option to the question, per @andrewdavidwong's note above?

Thx @andrewdavidwong! :)

**Comment by andrewdavidwong** [comment]
@hackerncoder: FYI, I just saw that three copies of your email to qubes-users were caught in the spam filter, so I allowed the most recent one through and rejected the other two duplicates.

**Comment by hackerncoder** [comment]
I could guess that is the problem, I think it's some trouble with DKIM. But thanks!

Yes, I added the +4yrs yesterday.

It *is* **really** hard. I'm a little child, and there is a month to Christmas.

**Comment by ninavizz** [comment]
@hackerncoder Heya! Would you mind re-enabling Replies in that thread? I'd love to engage the no-email-user person to contact the research@qubes-os email to interview them... as Qubes for journalists is my focus on a separate project. :D

**Comment by hackerncoder** [comment]
@ninavizz I can't. It's a mirror of qubes-users. I can tell you that their email address is: tetrahedra@danwin1210.me.

**Comment by hackerncoder** [comment]
@ninavizz I can't. It's a mirror of qubes-users (the mailling list). I can tell you that their email address is: tetrahedra [at] danwin1210 me.

**Comment by ninavizz** [comment]
@hackerncoder Awesome, TY!! Now that I've got it, would you mind nuking your 2 comments above, so the security of the individual behind that email is not connected to "a qubes user"?

**Comment by deeplow** [comment]
> @hackerncoder Heya! Would you mind re-enabling Replies in that thread? I'd love to engage the no-email-user person to contact the research@qubes-os email to interview them... as Qubes for journalists is my focus on a separate project. :D

@ninavizz we mirror the mailinglist on the forum. (I'm an admin there). So that's just a user-friendly read-only version of the mailinglist (hence why you can't reply). The user is named `no_email_user` because users only received the email forwarded from qubes-users@googlegroups.com so the forum software couldn't figure out the name as well. So I don't know how to reach out to the user directly (except maybe sending them a message on that mailinglist thread). But I'm no mailinglist guru.

**Comment by ninavizz** [comment]
@hackerncoder Hiya! How's this study going? If you'd feel up to sending me a .csv of the results, I'd LOVE to see them! email and gpg key are on the website... https://www.qubes-os.org/team/

**Comment by hackerncoder** [comment]
You're right @ninavizz! It's already been a month (wow). I'll get to them as soon as I can.

**Comment by hackerncoder** [comment]
@ninavizz. Something's wrong with your key on the website (or I don't understand how to use it. GPG is too powerfull.)

**Comment by hackerncoder** [comment]
@ninavizz Your key doesn't exist on gnupg, is there anywhere else I could get it?

**Comment by ninavizz** [comment]
@hackerncoder It should be on the [Qubes website](https://www.qubes-os.org/team/), (behind the GPG Key padlock-icon link), or on my own website at http://www.bigwheel.net behind a hyperlink at the bottom of the text blab on the first page. 

GPG is both wonderful and awful. I doubt you will ever find anyone to disagree with you, on that!

**Comment by hackerncoder** [comment]
The Qubes website links to http://keys.gnupg.net/pks/lookup?search=0x97A253361A54C66FE1DF515B081246AC3393E74F&fingerprint=on&exact=on&op=get&options=mr

![image](https://user-images.githubusercontent.com/14183017/114945252-460d2580-9e49-11eb-8fb4-0f8c46c213be.png)

**Comment by ninavizz** [comment]
> Seems the key on your website is for your gmail, and also served from an non-https site.

That is correct, because I am stupid—not you! "How many dorks does it take to screw-in a GPG key..." _sigh_

The key for team@research.qubes-os.org is on the Qubes Team page, and that's an email that both Marta and I check. Perhaps try that? 

@adw Not sure why the key issues are happening with my GMail key on the Qubes server. This has just been the week-o-follies for email, evidently. Will ping you off GH about that, tho.

**Comment by hackerncoder** [comment]
http://keys.gnupg.net/pks/lookup?search=0x4D2EB2FAE2BE5E8FD7852DE0DE1B6337FF6C36F1&fingerprint=on&exact=on&op=get&options=mr

![Nope](https://user-images.githubusercontent.com/14183017/114945252-460d2580-9e49-11eb-8fb4-0f8c46c213be.png)

&nbsp;

Hey, at least I got to look around your website. You're a great person, the FAQ, my interactions with you, just everything. You've given me some chuckles, some hope, I guess I really needed that. You indeed are "an upbeat optimist".

**Comment by ninavizz** [comment]
@hackerncoder LOL, ok—at least I'm accomplishing something? 👯‍♀️  I may not have yet pushed that key to the keyserver; ProtonMail manages that email address. Lemme go there to see if I did something wrong.

**Comment by ninavizz** [comment]
_insert andrew, marek, marta, michael, and all others, heartily facepalming_

**Comment by hackerncoder** [comment]
GPG is one beast, I can barely understand it.

**Comment by ninavizz** [comment]
@hackerncoder I DID IT!!! Try again? Apparently ProtonMail doesn't synchronize that for you, and I needed to add the key, my very own self... 
![giphy](https://user-images.githubusercontent.com/8262612/114953889-43410f00-9e0e-11eb-92a2-59471be7fbbe.gif)

**Comment by hackerncoder** [comment]
YAY! There is a key for team@research.qubes-os.org!

**Comment by hackerncoder** [comment]
Fits quite well, I think I figured mine out too; https://data.encryptionin.space/master.pub.pgp

**Comment by hackerncoder** [comment]
Aaaand Sent! Be sure to check your spam, I had some problems last time I tried to do mail: https://github.com/QubesOS/qubes-issues/issues/6463#issuecomment-800273829

**Comment by ninavizz** [comment]
@hackerncoder I promise I will never, ever send you proprietary formats! Will post some analysis here, within the week. Maybe tonight, but within the week. Dinnertime candy, yeah!

**Comment by adw** [comment]
> OH and @andrewdavidwong is not adw. Hi @adw, who... hasn't done anything since 2018, welp.

Hello! I wasn't expecting to be called out like this today, but hey, I've been busy :-)

**Comment by andrewdavidwong** [comment]
@ninavizz, you still need to upload your own key to public keyservers. You'll know you've succeeded when this link starts working: http://keys.gnupg.net/pks/lookup?search=0x97A253361A54C66FE1DF515B081246AC3393E74F&fingerprint=on&exact=on&op=get&options=mr

It is currently broken on the team page under your name.

**Comment by andrewdavidwong** [comment]
> This Issue is to contain feedback and findings to/from that effort.

That's not what qubes-issues is for. Please see:

https://www.qubes-os.org/doc/reporting-bugs/#issue-tracker-guidelines

At any rate, I think this issue has served its intended purpose. Closing.
